tutoroars/plugin.py

```diff
@@ -21,6 +21,7 @@
         # Each new setting is a pair: (setting_name, default_value).
         # Prefix your setting names with 'OARS_'.
         ("OARS_VERSION", __version__),
+        ("DOCKER_IMAGE_OARS", "python:3.8.16"),
```

Comment:
what if we try without the patch? so we always use the latest: https://hub.docker.com/layers/library/python/3.8